tests/php/src/Validation/URLValidationRESTControllerTest.php

```diff
@@ -5,7 +5,7 @@
  * @package AMP
  */
 
-namespace AmpProject\AmpWP\Tests;
+namespace AmpProject\AmpWP\Tests\Validation;
```

Comment:
Not related to the PR, but I also noticed that the namespace for this file was incorrect.